<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="videoPlayer">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>561</width>
      <height>481</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>10</y>
      <width>581</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="videoText">
       <property name="text">
        <string>视频路径</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_video"/>
     </item>
     <item>
      <widget class="QPushButton" name="openVideo">
       <property name="text">
        <string>打开视频</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>20</y>
      <width>54</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>检查结果</string>
    </property>
   </widget>
   <widget class="QTableView" name="CheckResult_tableView">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>40</y>
      <width>291</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startCheck">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>开始检测</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>videoPlayer</zorder>
   <zorder>label</zorder>
   <zorder>CheckResult_tableView</zorder>
   <zorder>startCheck</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
